resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -156,7 +156,7 @@ class="fas fa-info-circle"></i></a></b>
                         <button class="btn btn-success" type="submit">Save Profile</button>
                     </div>
                 </div>
-                {!! Form::close() !!}
+                {{ html()->form()->close() }}
```

Comment:
In reference to line 140 above: let's get rid of this comment especially since we aren't using the `Form::` syntax anymore

Reply:
Done